jekyll-dev.yml

```diff
@@ -220,17 +220,22 @@ konnect_saas: Kong Konnect
 package_name: Kong Konnect
 company_name: Kong Inc.
 mesh_product_name: Kong Mesh
+# Mesh specific config
 mesh_namespace: kong-mesh-system
 mesh_cp_name: kong-mesh-control-plane
-
-jekyll-generator-single-source:
-  versions_file: '_data/kong_versions.yml'
-  docs_nav_folder: '_data'
-
+mesh_ignored_links_regex:
```

Comment:
what are these new configs for?

Reply:
There are used in the Kuma plugins: https://github.com/kumahq/kuma-website/blob/master/jekyll-kuma-plugins/lib/jekyll/kuma-plugins/liquid/tags/jsonschema.rb#L60

They do 2 main things:

1. remove kong-mesh github links for auto-generated files
2. point where to find the auto-generated files